workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/RecoveryDecisionNode.java

```diff
@@ -69,13 +68,13 @@ public String getXML() {
             phn = previousHaltNode.getToNode().getName();
             LOGGER.debug("\n\n\n");
             LOGGER.debug("node entering in loop is " + previousHaltNode.getToNode().getName());
-            LOGGER.debug("ph_phn is " + ph_phn);
+            LOGGER.debug("ph_phn is " + phPhn);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ph_phn is " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)